File: rnapolis-frontend/src/app/entity/TeamMember.ts
export class TeamMember {
  name: string;
  surname: string;
  position: string;
  description: string;
  imagePath: string;

  constructor(name: string, surname: string, position: string, description: string, image: string){
    this.name = name;
    this.surname = surname;
    this.position = position;
    this.description = description;
    this.imagePath = image;
  }
}

File: rnapolis-backend/src/main/java/rnapolis/models/TeamMember.java
package rnapolis.models;

import java.time.Instant;
import lombok.Builder;
import lombok.Data;
import org.springframework.data.annotation.CreatedDate;
import org.springframework.data.annotation.Id;
import org.springframework.data.annotation.LastModifiedDate;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection = "teamMembers")
@Data
@Builder
public class TeamMember {

  @Id private String id;

  private String name;
  private String surname;
  private String position;
  private String description;
  private String imagePath;

  @CreatedDate private Instant createdDate;

  @LastModifiedDate private Instant lastModifiedDate;
}
